v1.0/pvp-specification.md

```diff
@@ -143,14 +143,14 @@ Now consider this scenario:
     version number bumped, as required by the PVP.
 2.  Package B, which can still work with the old and new version of
     package A, changes its dependency on package A to allow for both
-    versions. Package B only gets a patch-level bump.
-3.  Package C might or might not compile, depending on which patch-level
```

Comment:
This is incorrect change. 

Please read carefully through the example. In fact, here *patch-level bump* could be replaced with *patch-level bump or revision*.

Reply:
(Revisions are not part of PVP. Lets keep them out of it.)

Reply:
The problem I wanted to solve with this PR is that “patch-level” is not defined and hence confusing. Despite `D` being often used as SemVer-patch-level and Hackage having revision, neither are in the spec, hence the rewording.

Does it make sense, @phadej?

Reply:
Rewording doesn't make sense. Defined patch-level. Don't touch this example.

Reply:
> (Revisions are not part of PVP. Lets keep them out of it.)

FAQ could mention them nevertheless. I think similar change was in FAQ section,